changelog/36.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added a subpackage (`sunkit_image.flct`) for applying Fourier Local Coorelation Tracking
```

Comment:
Added a subpackage (`sunkit_image.flct`) for applying Fourier Local Correlation Tracking